src/modules/edittask.js

```diff
@@ -0,0 +1,34 @@
+import storeTask from './storetask.js';
+import { tasks } from './selector.js';
+
+const editTask = (e) => {
+  if (e.target.nodeName === 'LABEL') {
+    const taskDiv = e.target.closest('.task');
+    const task = tasks[taskDiv.getAttribute('id')];
+    const editInput = document.createElement('input');
+    editInput.value = task.description;
+    e.target.parentNode.replaceChild(editInput, e.target);
+    editInput.focus();
+
+    editInput.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter') {
+        task.description = editInput.value;
+        storeTask();
+        const newLabel = document.createElement('label');
+        newLabel.setAttribute('for', task.index);
+        newLabel.textContent = task.description;
+        editInput.parentNode.replaceChild(newLabel, editInput);
```

Comment:
Kindly make this a function so you can call it anytime you need to run the edit function. Repeating a code does make one codebase DRY.

Reply:
Thank you